src/codemagic/apple/app_store_connect/resource_manager.py

```diff
@@ -59,12 +63,16 @@ def __str__(self):
             restrictions = self._get_restrictions()
             if not restrictions:
                 return '*'
-            return ', '.join(f'{param}={shlex.quote(value)}' for param, value in restrictions.items())
+            return ', '.join(f'{param}={shlex.quote(str(value))}' for param, value in restrictions.items())
```

Comment:
need to ensure it's string since `expired` filter has boolean
```
>>> shlex.quote(True)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/stas/.pyenv/versions/3.8.0/lib/python3.8/shlex.py", line 324, in quote
    if _find_unsafe(s) is None:
TypeError: expected string or bytes-like object
>>> shlex.quote(str(True))
'True'
```